web_timeline/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Web timeline",
+    'summary': "Interactive visualization chart to show events in time",
+    "version": "10.0.1.0.0",
+    'author': 'ACSONE SA/NV, '
+              'Tecnativa, '
+              'Monk Software, ',
```

Comment:
You should remove the coma at the end of this line. It makes the module uninstallable.